cmd/kops/create_cluster.go

```diff
@@ -987,6 +1013,39 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 		return err
 	}
 
+	if c.DryRun {
+		var obj []runtime.Object
+		// if we do not set the ts we get creationTimestamp: null
```

Comment:
That's what kubectl does, so can you add a comment on the consequence of creationTimestamp null

Reply:
kubectl sets null, I think we should probably be doing the same